core/extensions/impl/crypto_extension.cpp

```diff
@@ -54,6 +57,26 @@ namespace kagome::extensions {
         : kVerifyFail;
   }
 
+  runtime::SizeType CryptoExtension::ext_sr25519_verify(
+      runtime::WasmPointer msg_data, runtime::SizeType msg_len,
+      runtime::WasmPointer sig_data, runtime::WasmPointer pubkey_data) {
+    // for some reason, 0 and 5 are used in the reference implementation, so
+    // it's better to stick to them in ours, at least for now
+    static constexpr uint32_t kVerifySuccess = 0;
+    static constexpr uint32_t kVerifyFail = 5;
```

Comment:
is that how it is done in polkadot?

Reply:
yeah, in case of success they return 5 :\